doc/1_applications.dox

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
Explained in the pull request comment. Doxygen includes files in alphabetical order after all documents after the one marked as the mainpage. So doing this allows me to control the order of these in the built-up documentation.